releasenotes/notes/data-streams-public-api-9655d5b0c7f6f26e.yaml

```diff
@@ -0,0 +1,3 @@
+features:
+  - |
+    data_streams: Adds public api to set produce and consume checkpoints.
```

Comment:
Should we mention the methods `set_produce_checkpoint` and `set_consume_checkpoint` here so that they're easy to look up in our codebade, if we're not going to document this more?